packages/config/index.js

```diff
@@ -4,34 +4,68 @@ var path = require('path');
 
 var root = require('pkg-dir').sync(process.cwd());
 
-var npmConfig = require('./lib/parse-env')('hops');
-if (!Object.keys(npmConfig).length) {
-  try {
-    require.resolve(path.join(root, 'package.json'));
-    npmConfig = require(path.join(root, 'package.json')).config.hops;
-  } catch (_) {}
+var explorer = require('cosmiconfig')('hops', {
+  rcExtensions: true,
+  stopDir: root,
+  sync: true,
+});
+
+function getRawConfig() {
+  var result = explorer.load(process.cwd());
+  return Object.assign(
+    {
+      https: false,
+      host: '0.0.0.0',
+      port: 8080,
+      locations: [],
+      basePath: '',
+      assetPath: '',
+      browsers: '> 1%, last 2 versions, Firefox ESR',
+      node: 'current',
+      envVars: { HOPS_MODE: 'dynamic' },
```

Comment:
not using a deep merge here means, that everyone who adds env vars must ensure to configure `HOPS_MODE`, isn't it?

Reply:
True. I somehow also feel we might have to go for `deep-assign`.